apps/app/middleware.ts

```diff
@@ -1,6 +1,65 @@
 import { authMiddleware } from '@repo/auth/middleware';
+import { env } from '@repo/env';
+import {
+  noseconeDefaults,
+  noseconeMiddleware,
+  withVercelToolbar,
+  type NoseconeOptions
+} from '@repo/security/middleware';
 
-export default authMiddleware();
+// Nosecone security headers configuration
+// https://docs.arcjet.com/nosecone/quick-start
+const noseconeOptions: NoseconeOptions = {
+  ...noseconeDefaults,
+  contentSecurityPolicy: {
+    ...noseconeDefaults.contentSecurityPolicy,
+    directives: {
```

Comment:
The included domains are based on the default configuration, but perhaps they should be conditional depending on on the configured env vars?

Reply:
If possible, I think disabling CSP rules by default is the best way to go, at least until #332 lands and we can figure out a way of composing CSP rules or something.

Reply:
Ok sure. I'll disable it and then add some docs explaining how to set it.